src/helpers/getCourseInfo.js

```diff
@@ -0,0 +1,27 @@
+import { getDuration } from './pipeDuration';
+import { dateGenerator } from './dateGenerator';
+import { getAuthors } from './getAuthors';
+import { mockedAuthorsList } from '../constants';
+
+export function getCourseInfo(props, params) {
+	const authorsList = mockedAuthorsList;
+	const courseInfoListOptions = [
+		{
+			param: 'authors',
+			content: ['Authors: ', getAuthors(props.authors, authorsList)],
+		},
+		{
+			param: 'duration',
+			content: ['Duration: ', `${getDuration(props.duration)} hours`],
+		},
+		{
+			param: 'created',
+			content: ['Created: ', dateGenerator(props.creationDate)],
+		},
+		{ param: 'id', content: ['ID: ', props.id] },
+	];
+	const courseInfo = params.map(
+		(p) => courseInfoListOptions.filter((info) => info.param === p)[0].content
```

Comment:
you can just simplify it with find, it does the same but shorter

Reply:
besides you can return directly, without extra variable coursesInfo because you don't use it anywhere else

Reply:
Right. Both suggestions applied!